doc-content/enterprise-only/openshift/template-overview-con.adoc

```diff
@@ -5,46 +5,46 @@
 
 ifdef::PAM[]
 ifeval::["{context}"=="openshift-authoring"]
-* `rhpam72-authoring.yaml` provides a {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. The {KIE_SERVER} uses an H2 database with persistent storage. You can use this environment to author processes, services, and other business assets. For details about this template, see <<rhpam72-authoring-ref>>.
+* `rhpam73-authoring.yaml` provides a {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. The {KIE_SERVER} uses an H2 database with persistent storage. You can use this environment to author processes, services, and other business assets. For details about this template, see <<rhpam73-authoring-ref>>.
 
-* `rhpam72-authoring-ha.yaml` provides a high-availability {CENTRAL}, a {KIE_SERVER} connected to the {CENTRAL}, and a MySQL instance that the {KIE_SERVER} uses. You can use this environment to author processes, services, and other business assets. The high-availability functionality is in technical preview. For details about this template, see <<rhpam72-authoring-ha-ref>>.
+* `rhpam73-authoring-ha.yaml` provides a high-availability {CENTRAL}, a {KIE_SERVER} connected to the {CENTRAL}, and a MySQL instance that the {KIE_SERVER} uses. You can use this environment to author processes, services, and other business assets. The high-availability functionality is in technical preview. For details about this template, see <<rhpam73-authoring-ha-ref>>.
```

Comment:
Minor suggestion - maybe changing this to a note to highlight that this is technology preview:

```
Note
Configuration of {CENTRAL} for high availability is currently technology preview.
```